health/REFERENCE.md

```diff
@@ -62,6 +62,7 @@ Netdata parses the following lines. Beneath the table is an in-depth explanation
 | [`hosts`](#alarm-line-hosts)                        | no              | Which hostnames will run this alarm.                                                  |
 | [`plugin`](#alarm-line-plugin)                      | no              | Restrict an alarm or template to only a certain plugin.                                             |
 | [`module`](#alarm-line-module)                      | no              | Restrict an alarm or template to only a certain module.                                             |
+| [`charts`](#alarm-line-charts)                      | no              | Restrict an alarm or template to specified charts.                                             |
```

Comment:
```suggestion
| [`charts`](#alarm-line-charts)                      | no              | Restrict an alarm or template to only certain charts.                                             |
```

to allign with other descriptions, sounds good?

Reply:
Also, should we make it clear that it works with templates only? 

> Restrict a template to only certain charts.

Reply:
I decided to call attention for this, because it does not make sense to use this feature with `alarm`.